consensus/service/src/config/tokens.rs

```diff
@@ -191,7 +313,7 @@ mod tests {
         assert_eq!(tokens, tokens2);
 
         // Validation should fail since MOB is not specified.
-        assert!(tokens.validate().is_err());
+        assert_validation_error(&tokens, "MOB token configuration not found");
```

Comment:
I switched to asserting specific error messages to protect from the case where something unrelated in the configuration changes (such as adding a new required field) which results in the validation failing, but not because of what the test is trying to verify.